helix-core/src/test.rs

```diff
@@ -147,6 +178,7 @@ pub fn plain(s: &str, selection: Selection) -> String {
     out
 }
 
+#[allow(clippy::module_inception)]
 #[cfg(test)]
 #[allow(clippy::module_inception)]
```

Comment:
Looks like this `allow` is duplicated

Reply:
Oh weird, that must have been a bad rebase. Will fix. Good catch